converters/fromZigbee.js

```diff
@@ -610,7 +619,13 @@ const converters = {
         cluster: 'msPressureMeasurement',
         type: ['attributeReport', 'readResponse'],
         convert: (model, msg, publish, options, meta) => {
-            const pressure = parseFloat(msg.data['measuredValue']);
+            let pressure = 0;
+            if (msg.data.hasOwnProperty('scaledValue')) {
```

Comment:
I see you added it for `scale`, but why not do the same for `scaledValue`?

Reply:
In my understanding this is like measured value, and can be sent without any other attributes

Reply:
Ah sorry, it's an alternative to `measureValue` indeed.